usertools/de_language_data.py

```diff
@@ -0,0 +1,72 @@
+# Export German Wiktionary language data to JSON.
```

Comment:
Languages JSON files are created from the code at here: https://github.com/tatuylonen/wiktextract/tree/master/languages

This code should be moved there called from the `get_data.py` file.